### PR TITLE
Update with changes from Dev

### PR DESCRIPTION
**Pull request by budak7273**

### CODE REVIEW

**Comment by budak7273** [comment]
CI is failing on the FRM repo, poked @porisius about it on discord

**Comment by budak7273** [comment]
Now CI is failing on the UE docs sites, going to try it another time

File: modules/ROOT/pages/Development/Cpp/hooking.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ Calling the scope need to have all the same parameters as the func hook signatur
 +
 [source,cpp]
 ----
-void hook(TCallScope<int(int)>& Scope, class* exampleClass, int exampleArg)
+void hook(TCallScope<int(int)>& Scope, class* exampleClass, int exampleArg) {
 	// stuff before final function call
 	int result = scope(exampleClass, exampleArg); // call following hooks (final function might get called as long as following hooks don't cancel/overwrite it)
```
Comment (mircearoata) [inline modules/ROOT/pages/Development/Cpp/hooking.adoc:47]:
```suggestion
	int result = Scope(exampleClass, exampleArg); // call following hooks (final function might get called as long as following hooks don't cancel/overwrite it)
```

File: modules/ROOT/pages/Development/Cpp/hooking.adoc [hunk 2]
```diff
@@ -52,7 +52,7 @@ void hook(TCallScope<int(int)>& Scope, class* exampleClass, int exampleArg)
 +
 [source,cpp]
 ----
-void hook(TCallScope<int(int)>& Scope, class* exampleClass, int exampleArg)
+void hook(TCallScope<int(int)>& Scope, class* exampleClass, int exampleArg) {
 	// final function might get called
 	scope.Override(customReturnValue);
```
Comment (mircearoata) [inline modules/ROOT/pages/Development/Cpp/hooking.adoc:57]:
```suggestion
	Scope.Override(customReturnValue);
```

File: modules/ROOT/pages/Development/Cpp/hooking.adoc [hunk 3]
```diff
@@ -33,7 +33,7 @@ The call scope object allows you to:
 +
 [source,cpp]
 ----
-void hook(TCallScope<...>& Scope, class* exampleClass, int exampleArg)
+void hook(TCallScope<...>& Scope, class* exampleClass, int exampleArg) {
  scope.Cancel();
```
Comment (mircearoata) [inline modules/ROOT/pages/Development/Cpp/hooking.adoc:37]:
```suggestion
 Scope.Cancel();
```

File: modules/ROOT/pages/ForUsers/SatisfactoryModManager.adoc [hunk 4]
```diff
@@ -78,8 +78,12 @@ for the rest of the steps.
...
 === Linux
 
-The latest Linux build of the mod manager can be downloaded from the
-https://github.com/satisfactorymodding/SatisfactoryModManager/releases/latest/download/Satisfactory-Mod-Manager.AppImage[github releases page].
+Linux builds of the mod manager can be found on the
+https://github.com/satisfactorymodding/SatisfactoryModManager/releases[GitHub releases page].
+You can directly download the latest release by clicking https://github.com/satisfactorymodding/SatisfactoryModManager/releases/latest/download/Satisfactory-Mod-Manager.AppImage[this link].
+
+// cspell:ignore Webkitgtk
+Webkitgtk is required - if you don't have it yet, obtain it through your package manager of choice.
```
Comment (mircearoata) [inline modules/ROOT/pages/ForUsers/SatisfactoryModManager.adoc:86]: This is not the case for SMM2, as it is an appimage so it bundles all dependencies, but is the case for SMM3.

```suggestion
`+webkit2gtk-4.0+` is required - if you don't have it yet, obtain it through your package manager of choice.
```

(cspell might complain about webkit2gtk, haven't checked)

**Review by mircearoata — CHANGES_REQUESTED** [review]

**Review by mircearoata — DISMISSED** [review]

**Review by mircearoata — APPROVED** [review]

**Pull request merged**